eng/Versions.props

```diff
@@ -180,20 +180,6 @@
          removed.  See https://github.com/dotnet/source-build/issues/2295 -->
     <MicrosoftBuildFrameworkVersion>15.7.179</MicrosoftBuildFrameworkVersion>
     <MicrosoftBuildUtilitiesCoreVersion>15.7.179</MicrosoftBuildUtilitiesCoreVersion>
-    <!--
-      Building .NET from source depends on one or two tar.gz files depending on the branch's current
-      source-buildability status.
-
-      PrivateSourceBuiltArtifactsPackageVersion is a tar.gz of .NET build outputs from a previous
-      build needed to build the current version of .NET. This is always defined, because .NET needs
-      to be bootstrappable at any point in time.
-
-      PrivateSourceBuiltPrebuiltsPackageVersion is a tar.gz of assets downloaded from the internet
-      that are needed to build the current version of .NET. Early in the lifecycle of a .NET major
-      or minor release, prebuilts may be needed. When the release is mature, prebuilts are not
-      necessary, and this property is removed from the file.
-    -->
-    <PrivateSourceBuiltArtifactsPackageVersion>7.0.102-4</PrivateSourceBuiltArtifactsPackageVersion>
```

Comment:
There is [logic](https://github.com/dotnet/installer/blob/release/7.0.1xx/src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets#L308) that references this that needs to be cleaned up.  It is possible someone could depend on the archive artifacts text file functionality.  This change would break them.  I'm not sure I have a strong opinion on this but wanted to bring it up.